code_generator/GdkCodeGenerator/src/Generation/Model/SchemaFunctionMappings.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using Improbable.CodeGeneration.Model;
+
+namespace Improbable.Gdk.CodeGenerator
+{
+    public class SchemaFunctionMappings
+    {
+        public static Dictionary<string, string> PrimitiveTypeToAddSchemaFunction = new Dictionary<string, string>
+        {
+            { BuiltInTypeConstants.builtInBool, "AddBool"},
+            { BuiltInTypeConstants.builtInBytes, "AddBytes" },
```

Comment:
Why is bytes a primative but not string?

Reply:
It is (see below) - this is a schema primitive (maybe built-in is a better name)

Reply:
Discussed offline - should possible change the name. These are proto 2 primitive types (also SchemObject getters) rather than Schemalang primitives.